v2.1/reference/tools/tikv-control.md

```diff
@@ -148,7 +148,9 @@ Use the `compact-cluster` command to manually compact data of the whole TiKV clu
 
 The `tombstone` command is usually used in circumstances where the sync-log is not enabled, and some data written in the Raft state machine is lost caused by power down.
 
-In a TiKV instance, you can use this command to set the status of some Regions to Tombstone. Then when you restart the instance, those Regions are skipped. Those Regions need to have enough healthy replicas in other TiKV instances to be able to continue writing and reading through the Raft mechanism. For example:
+In a TiKV instance, you can use this command to set the status of some Regions to Tombstone. Then when you restart the instance, those Regions are skipped. Those Regions need to have enough healthy replicas in other TiKV instances to be able to continue writing and reading through the Raft mechanism.
+
+Follow the two steps to set a Region to Tombstone:
```

Comment:
If we make it into steps, then we should make it clearer, more like steps (1. 2.). Refer to the Note below and consider taking out some content from it to make this section more like steps. You can first confirm it with the author.